packages/eyeglass/src/importers/ModuleImporter.ts

```diff
@@ -164,3 +155,13 @@ export default function ModuleImporter(eyeglass, sass, options, fallbackImporter
     }
   });
 }
+
+function defaultErrorHandler(done: (data: ImporterReturnType) => void) {
+  return function (err: Error | string) {
+    if (!(err instanceof Error)) {
+      err = new Error(err.toString());
```

Comment:
This approach to error wrapping is can loose information, also err could also be something which does not have a `toString()`

Reply:
Fixed by 921492d1b64a64499a49ce31665e3153ecb9dabc